src/components/course/course-header/tests/CourseImportantDates.test.jsx

```diff
@@ -0,0 +1,181 @@
+    date: now,
+    reason: 'subsidy_expired',
+  },
+},
+{
+  parentContentKey: null,
+  contentKey: 'edX+DemoX',
+  isAssignedCourseRun: false,
+  earliestPossibleExpiration: {
+    date: dayjs().add(20, 'days').toISOString(),
+    reason: 'subsidy_expired',
+  },
+}];
+const mockLearnerContentAssignments = {
+  allocatedAssignments: mockAllocatedAssignments,
+  hasAllocatedAssignments: mockAllocatedAssignments.length > 0,
+};
+
+const mockBaseRedeemablePolicies = {
+  redeemablePolicies: [],
+  expiredPolicies: [],
+  unexpiredPolicies: [],
+  learnerContentAssignments: {
+    assignments: [],
+    hasAssignments: false,
+    allocatedAssignments: [],
+    hasAllocatedAssignments: false,
+    acceptedAssignments: [],
+    hasAcceptedAssignments: false,
+    canceledAssignments: [],
+    hasCanceledAssignments: false,
+    expiredAssignments: [],
+    hasExpiredAssignments: false,
+    erroredAssignments: [],
+    hasErroredAssignments: false,
+    assignmentsForDisplay: [],
+    hasAssignmentsForDisplay: false,
+  },
+};
+
+const mockRedeemablePoliciesWithAllocatedAssignments = {
+  ...mockBaseRedeemablePolicies,
+  learnerContentAssignments: {
+    ...mockBaseRedeemablePolicies.learnerContentAssignments,
+    ...mockLearnerContentAssignments,
+  },
+};
+
+const mockCourseStartDate = dayjs().add(10, 'days').toISOString();
+
+const mockCourseRun = {
+  isEnrollable: true,
+  key: 'course-v1:edX+DemoX+2018',
+  pacingType: COURSE_PACING_MAP.SELF_PACED,
+  start: mockCourseStartDate,
+  availability: 'Current',
+  courseUuid: 'Foo',
+};
+
+const mockCourseMetadata = {
+  key: 'test-course-key',
+  subjects: [{
+    name: 'Test Subject 1',
+    slug: 'test-subject-slug',
+  }],
+  shortDescription: 'Course short description.',
+  title: 'Test Course Title',
+  owners: [TEST_OWNER],
+  programs: [],
+  image: {
+    src: 'http://test-image.url',
+  },
+  skills: [],
+  activeCourseRun: mockCourseRun,
+  courseRuns: [mockCourseRun],
+  availableCourseRuns: [mockCourseRun],
+};
+
+const CourseImportantDatesWrapper = () => (
+  <IntlProvider locale="en">
+    <AppContext.Provider value={{ authenticatedUser: mockAuthenticatedUser }}>
+      <CourseImportantDates />
+    </AppContext.Provider>
+  </IntlProvider>
+);
+
+describe('<CourseImportantDates />', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    useParams.mockReturnValue({ courseKey: 'edX+DemoX' });
+    useRedeemablePolicies.mockReturnValue({ data: mockBaseRedeemablePolicies });
+    useCourseMetadata.mockReturnValue({ data: mockCourseMetadata });
+  });
+  it('renders without crashing', () => {
+    useRedeemablePolicies.mockReturnValue({
+      data: mockRedeemablePoliciesWithAllocatedAssignments,
+    });
+
+    renderWithRouterProvider(<CourseImportantDatesWrapper />);
+
+    expect(screen.getByText('Important dates')).toBeTruthy();
+    expect(screen.getByText('Enroll-by date')).toBeTruthy();
+    expect(screen.getByText('Course starts')).toBeTruthy();
```

Comment:
nit: typically, the assertion with `screen.getByText` relies on `toBeInDocument()` vs. `toBeTruthy`?